### PR TITLE
Use datatables for more navigable image grids

### PR DESCRIPTION
**Pull request by oliverwm1**
Parts of prognostic reports with many 2D plots are hard to read/navigate. This PR uses [DataTables](https://datatables.net/) to make these pages easier to view and search.

Example report: https://storage.googleapis.com/vcm-ml-public/oliwm/example-datatables-report-v2/maps.html

### CODE REVIEW

File: workflows/diagnostics/fv3net/diagnostics/prognostic_run/views/matplotlib.py [hunk 1]
```diff
@@ -24,23 +24,29 @@
 template = jinja2.Template(
     """
 <h2> {{varfilter}} </h2>
-<table cellpadding="0" cellspacing="0">
+<table class="display" style="width:100%">
 
+<thead>
 <tr>
+<th>Variable</th>
```
Comment (brianhenn) [inline workflows/diagnostics/fv3net/diagnostics/prognostic_run/views/matplotlib.py:31]: Noah had a slack comment about making this bigger

File: workflows/diagnostics/fv3net/diagnostics/prognostic_run/views/matplotlib.py [hunk 2]
```diff
@@ -82,6 +89,7 @@ def plot_2d_matplotlib(
             logging.info(f"plotting {varname} in {run}")
             v = run_diags.get_variable(run, varname)
             long_name_and_units = f"{v.long_name} [{v.units}]"
+            variable_long_names[varname] = v.long_name
```
Comment (brianhenn) [inline workflows/diagnostics/fv3net/diagnostics/prognostic_run/views/matplotlib.py:92]: Nit: set this once instead of repeatedly?

**Review by brianhenn — COMMENTED** [review]
Thanks @oliverwm1 . See couple things below plus the slack discussion about metrics tables as datatables.

File: workflows/diagnostics/fv3net/diagnostics/prognostic_run/views/matplotlib.py (hunk 1 above)
Comment (oliverwm1) [reply workflows/diagnostics/fv3net/diagnostics/prognostic_run/views/matplotlib.py:31]: Increased size by 15%

File: workflows/diagnostics/fv3net/diagnostics/prognostic_run/views/matplotlib.py (hunk 2 above)
Comment (oliverwm1) [reply workflows/diagnostics/fv3net/diagnostics/prognostic_run/views/matplotlib.py]: Done. Added a `long_names` property to `RunDiagnostics`.

**Review by brianhenn — APPROVED** [review]
LGTM. The datatable formatting applies to tables in all of the pages right? Asking since the [example](https://storage.googleapis.com/vcm-ml-public/oliwm/example-datatables-report-v3/index.html) shows an html table on the process page.

**Comment by oliverwm1** [comment]
> LGTM. The datatable formatting applies to tables in all of the pages right? Asking since the [example](https://storage.googleapis.com/vcm-ml-public/oliwm/example-datatables-report-v3/index.html) shows an html table on the process page.

Yeah, I added the relevant script to the header of all the pages.

**Pull request merged**